test_scripts/Polices/Policy_Table_Update/010_ATF_P_TC_HMI_sends_GetURLs_one_app_registered.lua

```diff
@@ -0,0 +1,77 @@
+---------------------------------------------------------------------------------------------
+-- Requirements summary:
+-- In case HMI sends GetURLs and at least one app is registered SDL must return only default url and url related to registered app
+-- [HMI API] GetURLs request/response
+--
+-- Description:
+-- SDL should request PTU in case user requests PTU
```

Comment:
Please check description
Maybe it needs to be more like 

> In case HMI sends a request via SDL.GetURLs, PoliciesManager must respond with the list of pairs URLs taken from Local PT and appropriate internal appIDs (if exist) only for the applications being now connected to SDL.